feature-detects/elem/template.js

```diff
@@ -7,8 +7,8 @@
     "name": "HTML5Rocks Article",
     "href": "https://www.html5rocks.com/en/tutorials/webcomponents/template/"
   },{
-    "name": "W3 Spec",
-    "href": "https://dvcs.w3.org/hg/webcomponents/raw-file/tip/spec/templates/index.html"
+    "name": "W3C Spec",
+    "href": "https://web.archive.org/web/20170515055729/http://www.w3.org/TR/html5/scripting-1.html"
```

Comment:
just for info: when retrieving from web-archive-org did you take the last available useful date?

Reply:
I tried for most of them, but I I'm gonna double check.

Reply:
I think it isnt necessary to double check. That would take too much time for little gain.